artifacts/rhdh-plugins-reference/keycloak/keycloak-plugin-admin.adoc

```diff
@@ -6,17 +6,22 @@ The Keycloak backend plugin, which integrates Keycloak into {product-short}, has
 * Synchronization of Keycloak users in a realm.
 * Synchronization of Keycloak groups and their users in a realm.
 
+[NOTE]
+====
+The supported Keycloak version is `18.x`.
```

Comment:
If this is for 1.4, the supported version should be `24.x`.  For 1.3, it's `18.x`